pkgs/top-level/perl-packages.nix

```
@@ -16086,6 +16086,21 @@ let
     };
   };
 
+    RPM2 = buildPerlModule {
+    pname = "RPM2";
+    version = "1.4";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/L/LK/LKUNDRAK/RPM2-1.4.tar.gz";
+      sha256 = "5ecb42aa69324e6f4088abfae07313906e5aabf2f46f1204f3f1de59155bb636";
+    };
+    buildInputs = [ ModuleBuild pkgs.pkg-config pkgs.rpm ];
```

```suggestion
    buildInputs = [ pkgs.pkg-config pkgs.rpm ];
```
`buildPerlModule` adds it